knative-operator/pkg/common/util.go

```diff
@@ -31,3 +37,23 @@ func Configure(ks *servingv1alpha1.KnativeServing, cm, key, value string) bool {
 func IngressNamespace(servingNamespace string) string {
 	return servingNamespace + "-ingress"
 }
+
+// NamespaceFromIngressNamespace returns namespace after trim trailing "-ingress".
+func NamespaceFromIngressNamespace(ingressNamespace string) string {
+	return strings.TrimSuffix(ingressNamespace, "-ingress")
+}
+
+// KnativeServing gets KnativeServing in the specified namespace
+func KnativeServing(api client.Client, ksNamespace string) (types.NamespacedName, error) {
+	ksList := &servingv1alpha1.KnativeServingList{}
+	if err := api.List(context.TODO(), &client.ListOptions{Namespace: ksNamespace}, ksList); err != nil {
+		return types.NamespacedName{}, fmt.Errorf("failed to List KnativeServing: %w", err)
+	}
+	if len(ksList.Items) == 0 {
+		return types.NamespacedName{}, fmt.Errorf("KnativeServing is not found in namespace %q", ksNamespace)
+	}
+	return types.NamespacedName{
+		Namespace: ksList.Items[0].Namespace,
+		Name:      ksList.Items[0].Name,
+	}, nil
+}
```

Comment:
The changes to this file seem no longer needed now I think?

Reply:
Done.